qiskit/extensions/standard/rx.py

```diff
@@ -59,7 +59,31 @@ def to_matrix(self):
 
 
 def rx(self, theta, q):  # pylint: disable=invalid-name
-    """Apply Rx to q."""
+    """Apply Rx gate with angle theta to a specified qubit (q).
+    An Rx-gate implements a theta radian rotation of the qubit state vector about the
+    x-axis of the Bloch sphere.
+
+    Examples:
+
+        Circuit Representation:
+
+        .. jupyter-execute::
+
+            from qiskit import QuantumCircuit
+            import numpy
+
+            circuit = QuantumCircuit(1)
+            theta = numpy.pi/2
+            circuit.rx(theta,0)
+            circuit.draw()
+
+        Matrix Representation:
+
+        .. jupyter-execute::
+
+            from qiskit.extensions.standard.rx import RXGate
+            RXGate().to_matrix()
```

Comment:
From https://dev.azure.com/qiskit-ci/qiskit-terra/_build/results?buildId=6971&view=logs&j=23e88340-9b01-571b-4128-239215012d30&t=fd5b7ae3-9f70-5b67-b24e-b010b8098eb2&l=273

```suggestion
            RXGate(numpy.pi/2).to_matrix()
```